discovery-provider/src/tasks/entity_manager/entity_manager.py

```diff
@@ -333,8 +339,9 @@ def fetch_existing_entities(session: Session, entities_to_fetch: EntitiesToFetch
         )
     reposts: List[Repost] = session.query(Repost).filter(or_(*and_queries)).all()
     existing_entities[EntityType.REPOST] = {
+        # capitalize since repost_type is lower case and needs to match EntityType
         get_record_key(
-            repost.user_id, repost.repost_type, repost.repost_item_id
+            repost.user_id, repost.repost_type.capitalize(), repost.repost_item_id
```

Comment:
seems a little scary that we need to call capitalize here - is there no way to make consistent across the board

Reply:
yea i just moved it to the get_record_key util so that should help